src/views/BioPage.jsx

```diff
@@ -1,34 +1,95 @@
+  const currentIndex = bioData.findIndex(item => item.id === parseInt(id, 10));
+
+  const nextCards = bioData
+    .filter(
+      (item, index) =>
+        index !== currentIndex &&
+        index > currentIndex &&
+        index < currentIndex + 4,
+    )
+    .map(item => (
+      <SingleCard
+        key={item.id}
+        id={item.id}
+        image={item.image}
+        name={item.fullName}
+        body={item.occupation}
+      />
+    ));
+
+  const data = bioData[currentIndex];
+
+  const formatParagraphs = text => {
+    return text.split('\n').map((paragraph, index) => (
+      <p key={index} className="pb-4 text-lg">
+        {paragraph}
+      </p>
+    ));
+  };
 
   return (
-    <>
+    <div className="text-primary min-h-screen flex flex-col justify-center items-center mb-20">
       <div>
-        <div class="h-40 overflow-hidden bg-[#0B2E50]"></div>
-        <div class="mx-auto w-44 h-44 relative -mt-16 border-8 border-white rounded-full overflow-hidden">
+        <div className="h-52 bg-primary w-screen" />
+        <div className="mx-auto w-48 h-48 relative -mt-28 border-8 border-white rounded-full overflow-hidden">
           <img src={data.image} alt={data.fullName} />
         </div>
-        <div class="text-center mt-2">
-          <h2 class="text-3xl font-bold">{data.fullName}</h2>
-          <p class="text-gray-500">{data.occupation}</p>
+        <div className="text-center mt-2 pb-14">
+          <h1 className="text-3xl font-bold">{data.fullName}</h1>
+          <p>{data.occupation}</p>
         </div>
       </div>
+      <div className="max-w-screen-lg">
+        <div>
+          <div className="mb-10">
+            <div className="pb-4">
+              <h3 className="text-2xl font-bold">Who is {data.fullName}?</h3>
+              {formatParagraphs(data.bio.summary)}
+            </div>
+            <div className="pb-4">
+              <h3 className="text-2xl font-bold">
+                What is {data.name}'s impact?
+              </h3>
+              {formatParagraphs(data.bio.impact)}
+            </div>
+            <div className="pb-4">
+              <h3 className="text-2xl font-bold">
+                How did {data.name} conquer imposter syndrome?
+              </h3>
+              {formatParagraphs(data.bio.career)}
+            </div>
+          </div>
 
-      <section>
-        <h3 className="text-xl font-bold">Who is {data.fullName}?</h3>
-        <p>{data.bio.summary}</p>
-        <h3 className="text-xl font-bold">What is {data.name}'s impact?</h3>
-        <p>{data.bio.impact}</p>
-        <h3 className="text-xl font-bold">
-          How did {data.name} conquer imposter syndrome?
-        </h3>
-        <p>{data.bio.career}</p>
-      </section>
-    </>
+          <div className="mb-14">
+            <h2 className="text-2xl font-bold">
+              {data.name}'s Suggested Resources
+            </h2>
+            <div className="grid md:grid-cols-2">
+              {data.resources.map(item => (
```

Comment:
To make the code consistent, I think you can move this logic up like how you created the nextCards.

Reply:
We should use optional chaining here to make sure the app doesn't throw an error if there are no resources. Though I'm not sure if this will also take care of `resources` being defined as an empty array - either way it's good practice:
`data.resources?.map`

Reply:
Doing `.map()` on empty array will not produce an error, but will return an empty array. An  empty array is a renderable item in react and won't produce error in` render() `and will not render anything because it's empty. You can use `.length` check `data.resources.length > 0` which is more reliable.